### DISCUSSION TITLE
Skip uploading empty files

### DISCUSSION THREAD

**Discussion by CraigSiemens** [crowdin/github-action#151] [Q&A] [answered]
We're working on adding this to our project and are getting the following error.

```
❌ Failed to upload the '/github/workspace/fastlane/metadata/en-CA/promotional_text.txt' file to the storage. Please contact our support team for help
❌ Wrong parameters: 
<key: fileStream, code: streamIsEmpty, message: Stream size is null. Not empty content expected>
```

We use fastlane for sending metadata to the appstore, but some of the files can be empty since they're optional. 

Is there an option for this action to skip uploading empty files? Or is there another workaround that we could use?

**Comment by andrii-bodnar** [comment]
Hi @CraigSiemens, thanks for your request!

Probably, we should more gracefully handle this case on the CLI side and throw a non-critical warning about the empty file 🤔

**Reply by andrii-bodnar** [reply]
PR: https://github.com/crowdin/crowdin-cli/pull/517

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @CraigSiemens!
The empty files handling was improved in the recent release - [1.5.3](https://github.com/crowdin/github-action/releases/tag/1.5.3)
Now it will show a non-blocking warning and the command shouldn't fail in this case
